app/src/main/res/values/strings.xml

```diff
@@ -1016,6 +1016,17 @@ If there are no signs along the whole street which apply for the highlighted sec
     <string name="quest_maxweight_answer_other_sign">Sign looks different…</string>
     <string name="quest_maxweight_unsupported_sign_request_photo">Would you like to leave a note? A note with a photo will enable other mappers to take care of it.</string>
 
+    <string name="quest_memorialType_title">What kind of memorial is this?</string>
+    <string name="quest_memorialType_statue">Statue</string>
+    <string name="quest_memorialType_bust">Bust</string>
+    <string name="quest_memorialType_plaque">Plaque</string>
+    <string name="quest_memorialType_war_memorial">War memorial</string>
+    <string name="quest_memorialType_stone">Stone</string>
+    <string name="quest_memorialType_obelisk">Obelisk</string>
+    <string name="quest_memorialType_stele_wooden">Wooden stele (memorial slab)</string>
+    <string name="quest_memorialType_stele_stone">Stone stele (memorial slab)</string>
```

Comment:
```suggestion
    <string name="quest_memorialType_obelisk">Obelisk (pointy pillar)</string>
    <string name="quest_memorialType_stele_wooden">Wooden stele (flat slab)</string>
    <string name="quest_memorialType_stele_stone">Stone stele (flat slab)</string>
```

to distinguish obelisk/pillar. "memorial" word can be removed, as the whole quest is about memorials.

Reply:
I \*think\* that for obelisks being four sided is also important, pointed round column is not an obelisk.

Reply:
maybe `memorial slab, flat` would be better to avoid tautology?

Reply:
`memorial slab, flat` seems fine to me, I was trying to shorten it down. 

As for obelisks being 4 sided, it definitely seems to be most common case by vast majority (`4-sided pointy pillar`?), but [wikipedia seems to list other shapes too as Obelisks](https://en.wikipedia.org/wiki/List_of_modern_obelisks) (e.g. [Killiney Hill](https://en.wikipedia.org/wiki/Killiney_Hill) Obelisk, [Heroes Monument](https://en.wikipedia.org/wiki/Heroes_Monument), [Monument to the abolition of slavery](https://en.wikipedia.org/wiki/Monumento_a_la_abolici%C3%B3n_de_la_esclavitud), [Bayonet-Obelisk of the War Memorial](https://upload.wikimedia.org/wikipedia/commons/thumb/b/b5/Brest_Brest_Fortress_9085_2150.jpg/100px-Brest_Brest_Fortress_9085_2150.jpg), [Colonna Mediterranea](https://en.wikipedia.org/wiki/Colonna_Mediterranea) etc.)

Reply:
For 4-sided: I opened https://en.wikipedia.org/wiki/Talk:Obelisk#Four_sided%3F as I am not 100% sure